packages/agents/cartographer/poller/src/lib/operations/transfers.ts

```diff
@@ -21,7 +21,7 @@ const getMaxReconcileTimestamp = (transfers: XTransfer[]): number => {
     : Math.max(...transfers.map((transfer) => transfer.destination?.reconcile?.timestamp ?? 0));
 };
 
-const getMaxTimestamp = (entities: SlippageUpdate[]): number => {
+const getMaxTimestamp = (entities: any[]): number => {
```

Comment:
Assuming build doesn't like this

Reply:
This is fine actually because it's a generic function.